### ISSUE TITLE
Why cant i repair my weapons?

### ISSUE THREAD

**Issue by ZombieApp** [pWn3d1337/Techguns#268]
Please Help me if u can i cant find a way to

**Comment by Mooshroom8** [comment]
Yeah So if you are talking about a gun, it doesn't have durability. That's the ammo, and if you have anything else like power tools: More Ammo. Anything else is vanilla except combat knife.
